meta-koi/recipes-core/psplash/psplash_git.bbappend

```diff
@@ -0,0 +1,6 @@
+SRC_URI:append:koi = " file://0002-Disable-double-buffering.patch"
```

Comment:
koi does a show a boot logo. It does, however, seem to behave in a slightly different way to other watches.

Reply:
the reason I wrote this initially is that the logo seemed to only show when I had the display driver set up correctly. This was probably just a coincidence.
Another quirk is that the system spends ages at the casio boot logo, and only shows the psplash asteroid logo for a split second before the launcher takes over. Perhaps this points to some inefficiencies in the early boot process, but the overall boot-to-ui time seems comparable to other watches, so I'm not making it a priority. 

back on topic: I'm removing this and marking this as resolved, since I've removed this file. behaviour is identical without it.